app/kuma-dp/pkg/dataplane/accesslogs/v3/handler.go

```diff
@@ -1,43 +1,11 @@
 package v3
 
-import (
-	envoy_accesslog "github.com/envoyproxy/go-control-plane/envoy/service/accesslog/v3"
-	"github.com/pkg/errors"
-
-	accesslog "github.com/kumahq/kuma/pkg/envoy/accesslog/v3"
-)
-
 type handler struct {
```

Comment:
do we even need handler anymore if that just delegates to sender? Can we just use sender in places when we use handler currently?